printer_mods/JosAr/Klicky-Probe/README.md

```diff
@@ -15,101 +15,372 @@ The objectives for this project are:
 
 Would also like to thank the Voron discord community and VoronDesign for all the work that was and still is being made to maintain the Voron ecosystem.
 
-The probe accuracy output is something like this:
-probe accuracy results: maximum 6.430000, minimum 6.426250, range 0.003750, average 6.428750, median 6.428750, standard deviation 0.000791
+
+
+### klipper Configuration and Probe offset
+
+Download the appropriate Klicky Probe.cfg and upload it to your klipper Config folder.
+
+Then open your printer.cfg file and add the following line before the "Macros" Section.
+
+`[include Klicky Probe.cfg]`
+
+Within your printer.cfg file search for the `[probe]` section and change the pin assignment to the new one `z:P1.28` or the ID where you connected your Klicky Probe to. Depending on your switch you may need to add a `!` to invert that pin (normally open vs. normally closed).
+
+Within the `[probe]` section also adjust your probe offset to the following values.
+
+You need to set the probe offset within your `printer.cfg`  
 
 There is now an arrow on the probe telling you where should the switch pole be to have the correct offset.
 The probe offsets are:
-- z_offset = 6.42
-- x_offset: 0
-- y_offset: 19.75
+```python
+z_offset = 6.42
+x_offset: 0 
+y_offset: 19.75
+```
+
+### Z endstop and Probe configuration
+
+If you want to use the Klicky Probe as your Z endstop, you need to change the `endstop_pin:` under the `[stepper_z]` section to `probe:z_virtual_endstop` . Just comment out the old one and add a new line `endstop_pin: probe:z_virtual_endstop`. 
+
+You don't need to change anything else in the printer.cfg, so leave it as it is, you can easily revert back to the original setup.
+
+### Use Klicky Probe with/without Z endstop switch (Voron)
+
+If you want to use the Z endstop switch of the Voron you also need to set the following two lines, this is the Z endstop Location from your `printer.cfg`.
+
+```python
+variable_z_endstop_x:
+variable_z_endstop_y:
+```
+
+If you want to use your Klicky Probe as a Z endstop, then you need to set the two lines to, `0`.
+
+```python
+variable_z_endstop_x:     0
+variable_z_endstop_y:     0
+```
+
+### Adjust Probe Pickup Position
+
+One of the last things we need to do is to adjust the probe pickup position.
+
+For this we need to make sure that the gantry is [deracked](https://www.youtube.com/watch?v=cOn6u9kXvy0), the x and y axis are homed and the probe is manually attached to the AB-Mount.
+
+Now manually move the toolhead to the probe dock and move it so far to the back that the probe docks, note the Y-Position.
+Next, again manually, move the toolhead left and right until the probe it is perfectly aligned with the mount, note the X.Position.
+
+Open you `Klicky Probe.cfg` and find the `#dock location` section and edit the following two line
```

Comment:
Replace `you` with `your`.
Replace `Klicky Probe.cfg` with `klicky-probe.cfg`
Replace `two line` with `two lines`